cpp/open3d/t/geometry/kernel/ImageImpl.h

```diff
@@ -41,6 +41,79 @@ using std::isinf;
 using std::isnan;
 #endif
 
+#ifdef __CUDACC__
+void ToCUDA
+#else
+void ToCPU
+#endif
+        (const core::Tensor& src,
+         core::Tensor& dst,
+         double scale,
+         double offset) {
+    NDArrayIndexer src_indexer(src, 2);
+    NDArrayIndexer dst_indexer(dst, 2);
+    core::Dtype dst_dtype = dst.GetDtype();
+    const int64_t rows = src.GetShape(0);
+    const int64_t cols = src.GetShape(1);
+    const int64_t channels = src.GetShape(2);
+
+#define LINEAR_SATURATE(elem_t)                                                \
+    elem_t limits[2] = {std::numeric_limits<elem_t>::min(),                    \
+                        std::numeric_limits<elem_t>::max()};                   \
+    calc_t c_scale = static_cast<calc_t>(scale);                               \
+    calc_t c_offset = static_cast<calc_t>(offset);                             \
+    DISPATCH_DTYPE_TO_TEMPLATE(src.GetDtype(), [&]() {                         \
+        core::ParallelFor(                                                     \
+                src.GetDevice(), rows* cols,                                   \
+                [=] OPEN3D_DEVICE(int64_t workload_idx) {                      \
+                    int64_t col = workload_idx % cols;                         \
+                    int64_t row = workload_idx / cols;                         \
+                    auto src_ptr = src_indexer.GetDataPtr<scalar_t>(col, row); \
```

Comment:
Essentially, we are doing `cpp/open3d/core/kernel/UnaryEWCPU.cpp::LaunchUnaryEWKernel()`. In the future, we can expose this such that it can be used for custom element-wise kernels.

Reply:
Done.